ui/app/components/app/permissions-connect-footer/index.scss

```
@@ -5,9 +5,8 @@
   align-items: center;
 
   &__text {
-    @extend %content-text;
+    @extend %H6;
```

This should be H7, which would match the deleted font size (12px)
```suggestion
    @extend %H7;
```